src/@types/InteractionContext.d.ts

```diff
@@ -213,6 +223,12 @@ declare global {
 		readonly targetMessage: Message;
 	}
 
+	/** Information relevant to button presses */
+	interface ButtonContext extends InteractionContext {
+		component: APIButtonComponent;
```

Comment:
Please `APIButtonComponent`.

Reply:
Thanks, I'll be careful to disable `skipLibCheck` to check this kind of thing in the future.